content/components/action-menu.mdx

```diff
@@ -11,41 +11,231 @@ export default ComponentLayout
+  trigger, a label must be persist either internally or externally.
+</Caption>
+
+<img
+  src="https://user-images.githubusercontent.com/18661030/226486463-893231bb-8f57-4255-83a4-7d468b2be5ed.png"
+  width="960"
+  alt=""
+/>
+<Caption>Menus can contain simple actions and links, single select items, or multi select items.</Caption>
+
+## Behavior
+
+### Opening
+
+<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+  <div>
+    <img
+      src="https://user-images.githubusercontent.com/18661030/226486479-52730f0f-cc41-41e0-a996-a974899cbc4e.png"
+      alt=""
+    />
+    <Caption>Open by clicking the trigger button with a mouse</Caption>
+  </div>
+  <div>
+    <img
+      src="https://user-images.githubusercontent.com/18661030/226486480-4a807f86-b2b0-4d12-8a43-8cc95d36d995.png"
+      alt=""
+    />
+    <Caption>Open by focusing the trigger button and hitting return, space, or any arrow key</Caption>
+  </div>
 </Box>
-</div>
-<div>
-<Link href="https://primer.style/react/ActionMenu#with-selection">
-<img alt="Action Menu with selection" src="https://user-images.githubusercontent.com/293280/123881067-cdbaea00-d8f8-11eb-98e4-e57c64489308.png" />
-</Link>
-</div>
-<div>
-<Heading sx={{fontSize: 3}}>To make one or more selections</Heading>
-<Box as="p">
-
-Action menu can be used for making a selection among a small list of options. The list is usually predefined with system values, but in some cases it can include user-provided data when those are known not to grow into too many items.
 
+### Closing
+
+#### Default behavior
+
+<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+  <div>
+    <img
+      src="https://user-images.githubusercontent.com/18661030/226486482-fc457572-342d-42d3-9da6-fa87ef009cea.png"
+      alt=""
+    />
+    <Caption>
+      Close the menu by clicking on an item, clicking on the trigger, or clicking anywhere outside the menu.
+    </Caption>
+  </div>
+  <div>
+    <img
+      src="https://user-images.githubusercontent.com/18661030/226486483-e3d658ea-4148-47d6-95ed-5ca90cc55b38.png"
+      alt=""
+    />
+    <Caption>Close the menu by hitting enter on an item, or esc.</Caption>
+  </div>
 </Box>
-</div>
+
+#### Single and multi select
+
+<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+  <div>
+    <CustomVideoPlayer
+      width="100%"
+      src="https://user-images.githubusercontent.com/18661030/226487362-b66daa5f-40b3-4f73-908c-19a74cabd435.mp4"
+    />
+    <Caption>Single select menu closes upon selection and may update the button label.</Caption>
+  </div>
+  <div>
+    <CustomVideoPlayer
+      width="100%"
+      src="https://user-images.githubusercontent.com/18661030/226487367-680395c7-1859-40a0-bbfc-53f37a686662.mp4"
+    />
+    <Caption>Multi select menu closes upon selection and can be re-opened to select more options.</Caption>
```

Comment:
oh nice, so the menu doesn't need to stay open anymore on selection? we were waiting on outcome of this decision for Primer Brand's action menu, so we can go ahead and implement this now I guess.

Reply:
That was @ichelsea's recommendation, so I would feel confident moving forward with it!